crates/polars-io/src/cloud/options.rs

```diff
@@ -90,8 +90,8 @@ impl FromStr for CloudType {
     fn from_str(url: &str) -> Result<Self, Self::Err> {
         let parsed = Url::parse(url).map_err(to_compute_err)?;
         Ok(match parsed.scheme() {
-            "s3" => Self::Aws,
-            "az" | "adl" | "abfs" => Self::Azure,
+            "s3" | "s3a" => Self::Aws,
+            "az" | "azure" | "adl" | "abfs" | "abfss" => Self::Azure,
             "gs" | "gcp" | "gcs" => Self::Gcp,
```

Comment:
`gcp` isn't in the `is_cloud_url` util - is it a valid prefix? If so, the utils should be updated to include this. If not, should be removed here.

Reply:
I don't see this at `https://docs.rs/object_store/latest/object_store/gcp/struct.GoogleCloudStorageBuilder.html#method.with_url`... So I doubt it's valid. I don't even think the other 2 are, but I'm a bit warry to remove those.
For fsspec I find this: https://filesystem-spec.readthedocs.io/en/latest/api.html?highlight=google#other-known-implementations
[gcsfs](https://gcsfs.readthedocs.io/en/latest/) for Google Cloud Storage
[gdrive](https://github.com/fsspec/gdrivefs) to access Google Drive and shares (experimental)

What do you think is the approach we should take here?

Reply:
Hm... let's leave it for now. Indeed we should probably be using [`object_store::parse_url`](https://docs.rs/object_store/latest/object_store/fn.parse_url.html) anyway...